main/zoe/guide/contracts/covered-call.md

```diff
@@ -115,23 +114,21 @@ const daveSwapProposal = harden({
 
 const daveSwapPayments = harden({ Price: daveBucksPayment });
 
-const { seat: daveSwapSeat, payout: daveSwapPayoutP } = await zoe.redeem(
+const { outcome, payout: daveSwapPayoutP } = await E(zoe).offer(
   daveSwapInvite,
   daveSwapProposal,
   daveSwapPayments,
 );
 
-// Dave completes the swap and awaits his payout
-const daveSwapOutcome = await daveSwapSeat.matchOffer();
-
 const daveSwapPayout = await daveSwapPayoutP;
 const daveOption = await daveSwapPayout.Asset;
 const daveBucksPayout = await daveSwapPayout.Price;
 ```
 
 ## Exercising the Option
 
-Now that Dave owns the option he can exercise the option by making and completing a proposal to the covered call. First, he escrows with Zoe, then he calls the `exercise` method to complete the proposal:
+Now that Dave owns the option he can exercise the option. He can do by
+redeeming the option (an invite) and making an offer:
```

Comment:
"...he can exercise the option by submitting an offer that pays the required exercise price in exchange for the underlying asset:"